stripe-ui-core/src/test/java/com/stripe/android/uicore/elements/PhoneNumberFormatterTest.kt

```diff
@@ -8,7 +8,7 @@ internal class PhoneNumberFormatterTest {
 
     @Test
     fun `Phone number is correctly formatted for US locale`() {
-        val formatter = PhoneNumberFormatter.forCountry("US")
+        val formatter = PhoneNumberFormatter.forCountry("US") // "(###) ###-####"
```

Comment:
Do we test every phone number that we support? If not, can we add tests for them? Do you think it is valuable to add tests for each country?

Reply:
hmm what we changed is just limiting the length of converted phone number, which is covered by the test, not sure adding a single test for each region adds too much value here, instead I added another test for a customized pattern